packages/design-tokens/scripts/style-dictionary.js

```diff
@@ -0,0 +1,495 @@
+/**
+ * Ported from primer/primitives, as it was removed in v8 release and we still need it
+ */
+
+const glob = require('fast-glob')
+const StyleDictionary = require('style-dictionary')
+
+const {fileHeader, formattedVariables} = StyleDictionary.formatHelpers
+
+//-----
+// functions to be extracted
+// TODO: extract to a separate files
+
+const pathToKebabCase = token => token.path.join('-')
+
+const pathToDotNotation = token => token.path.join('.')
+
+const capitalize = string => string[0].toUpperCase() + string.slice(1)
+
+const pathToPascalCase = token => token.path.map(tokenPathItems => capitalize(tokenPathItems)).join('')
+
+// REGISTER THE CUSTOM TRANFORMS
+
+/**
+ * transform: scss variable names
+ * example: `$namespace-item-variant-property-modifier`
+ */
+StyleDictionary.registerTransform({
```

Comment:
You should be able to use this: https://github.com/primer/primitives/blob/main/src/transformers/namePathToKebabCase.ts

Reply:
🙌 nice.. 

added as a follow up PR item